src/contexts/useActiveVideo.tsx

```diff
@@ -0,0 +1,42 @@
+import { useCallback } from 'preact/compat'
+import useSdkConfigurationService from '~contexts/useSdkConfigurationService'
+import { StepConfig } from '~types/steps'
+
+const useActiveVideo = () => {
+  const { experimental_features } = useSdkConfigurationService()
+
+  const enabled = experimental_features?.enable_liveness_experiment ?? false
```

Comment:
`enable_liveness_experiment` is deprecated, lets use `experimental_features?.motion_experiment?.enabled` which is the new one